src/components/SelectedMovie/SelectedMovie.js

```diff
@@ -73,7 +75,9 @@ class SelectedMovie extends Component {
           <div className="movie-info">
           <article className="overview">
               <h3>{overview}</h3>
-              <button className="return-home" onClick={() => this.returnHome()}>Return Home</button>
+          <Link to={"/"}>
+              <button className="return-home">Return Home</button>
```

Comment:
This is brilliant! Good catch placing link around this button. It removes the need for any click handlers!!!